swagger-spec/actions/definition.yaml

```diff
@@ -172,7 +172,7 @@ properties:
                 example: "osf"
               type:
                 type: string
-                example: "providers"
+                example: "preprint-provider"
```

Comment:
Is it not plural?

Reply:
I believe they're meant to be plural, yes, so `preprint-providers` sounds the most right to me.

Reply:
Ok, just so aren't messing up our own docs with singular `type`s, I'm going to make a new PR to update this